code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -106,6 +106,25 @@
 	item_state = "firstaid-internalbleed"
 	items_to_spawn = list(/obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector = 4)
 
+/obj/item/weapon/storage/firstaid/vox
+	name = "oxygen deprivation first-aid kit"
```

Comment:
alternatively, 
```suggestion
	name = "oxygen exposure first-aid kit"
```

Reply:
Oops, I don't know how I missed that. I like this suggestion more